### ISSUE TITLE
Fragment selectors or fragment-URI shortcuts when annotating images onto canvas?

### ISSUE THREAD

**Issue by zimeon** [IIIF/api#1338]
In Web Annotation, [fragment selectors](https://www.w3.org/TR/annotation-model/#fragment-selector) are the RECOMMENDED way to reference a fragment. Should this be the IIIF RECOMMENDED way to paint onto a portion of the canvas (and use fragments in annotations in other places)? If so then I'd assume a direct URI-with-fragment-ed would also be allowed (per Web Annnotation) _OR_ Is this all too long winded, we we instead profile Web Annotation to say that the short URI-with-fragment-ed form SHOULD or MUST be used?

Taking a short version of the second example from http://iiif.io/api/presentation/2.1/#segments:

```
{
  "@context": ".../presentation/2/context.json",
  "@id": "http://ex.org/anno1",
  "@type": "oa:Annotation",
  "motivation": "sc:painting",
  "resource": {
    "@id": "http://ex.org/image.jpg",
    ...
   },
  "on": "http://ex.org/canvas/p1#xywh=0,0,600,900"
}
```

we would in Presentation 3 have the following if we follow the RECOMMENDED fragment selector form:

```
{
  "@context": [
    "http://www.w3.org/ns/anno.jsonld",
    ".../presentation/3/context.json"
  ],
  "@id": "http://ex.org/anno1",
  "@type": "Annotation",
  "motivation": "painting",
  "body": {
    "@id": "http://ex.org/image.jpg",
    ...
   },
  "target": {
    "source": "http://ex.org/canvas/p1",
    "selector": {
      "type": "FragmentSelector",
      "conformsTo": "http://www.w3.org/TR/media-frags/",
      "value": "xywh=0,0,600,900"
    }
  }
}
```

**Comment by azaroth42** [comment]
Impacts #1332 as the fragment selector would be more compatible with the proposed AudioTrackSelector as well as the regular SvgSelector.

**Comment by azaroth42** [comment]
Consensus on AV call of 2018-02-06, and Editors on 2018-02-07 was that the selector model should be recommended but not required, and the fragment URI pattern documented as also possible.

**Comment by azaroth42** [comment]
I think this needs further discussion in light of #1386

**Comment by tomcrane** [comment]
Use the full form in examples, explain that the hash-fragment URI form _MAY_ be used. But where in the spec does this explanation go?

**Comment by aisaac** [comment]
>But where in the spec does this explanation go?

In 5.5 Annotations?

**Comment by azaroth42** [comment]
Given that we're only talking about Canvas annotations, and we have #1386, I think they it can go into Canvas.

See the (currently not written, but a bright place holder) block in:
http://preview.iiif.io/api/airplane/api/presentation/3.0/#canvas

**Comment by azaroth42** [comment]
Closed by #1500
